syncer/optimist.go

```diff
@@ -27,11 +27,12 @@ import (
 	"github.com/pingcap/dm/pkg/terror"
 )
 
-// trackedDDL keeps data needed for schema tracker.
-type trackedDDL struct {
-	rawSQL string
-	stmt   ast.StmtNode
-	tables [][]*filter.Table
+// trackInfo keeps data needed for schema tracker.
+type trackInfo struct {
```

Comment:
seems `trackInfo`  is used not only in `handleQueryEventOptimistic` but also used in other place, so maybe move this struct to `ddl.go` from `optimsit.go`?